meta/fraction-metadata/src/main/java/org/wildfly/swarm/fractions/scanner/FilePresenceScanner.java

```diff
@@ -22,12 +22,13 @@
 import java.util.function.Consumer;
 
 import org.wildfly.swarm.spi.meta.FileDetector;
+import org.wildfly.swarm.spi.meta.FileSource;
 import org.wildfly.swarm.spi.meta.FractionDetector;
 
 /**
  * @author Heiko Braun
  */
-public class FilePresenceScanner implements Scanner<String> {
+public class FilePresenceScanner implements Scanner<FileSource> {
     @Override
     public String extension() {
         return XML;
```

Comment:
No no no, this question has nothing to do with your changes. I'm just wondering what's your opinion about `FilePresenceScanner.extension` always returning `"xml"`. I was thinking it might be overly restrictive if we ever detect on something else than just XML files, but it's probably a premature generalization on my side :-)